### DISCUSSION TITLE
Unable to paste DLC blocks in both Offline and Online worlds

### DISCUSSION THREAD

**Discussion by Pepper-Man** [Lamer87/Space_Engineers_DLC_unlocker#44] [Q&A] [answered]
This unlocker works pretty well, I am able to manually place the blocks from DLCs I don't own, however I cannot paste them from workshop blueprints. That's kinda all I wanted to use this for - I understand that it doesn't work in online servers, but even when just loading local worlds set to Offline, it still gives me the warning that the blueprint contains blocks from DLCs I don't own, and doesn't paste them.

**Comment by Lamer87** [comment, marked as answer]
Unfortunately nothing can be done, there is a steam check which cannot be avoided in any way (same for armor skins).
The only thing you can do is wait for the steam discounts, and buy dlcs at a ridiculous price.

**Comment by Pepper-Man** [comment]
Ah well that's a shame. Perhaps consider amending the README, as its a little misleading at the moment. It only mentions 
> It is not possible to paste DLC items contained in projects when playing _on a server._

**Comment by Lamer87** [comment]
You're right, corrected the mistake.
Switching from issue to Discussion.

**Comment by jozsaVilmos** [comment]
I've found a method to paste workshop blueprint with unlocked DLCs. 

Use this mod and build the ship. DLC blocks will be added. I suspect the mods spawns them so it's the same check as building not like projecting.

[Instant Projector](https://steamcommunity.com/sharedfiles/filedetails/?id=2096677783)

@Lamer87 maybe add to the Readme

**Reply by Lamer87** [reply]
Done, thanks :)
